content/authors/ehis-edemakhiota/index.md

```diff
@@ -2,4 +2,8 @@
 title: Ehis Edemakhiota
 type: authors
 ---
-Ehis is a Software Engineer. He is passionate about problem-solving. He enjoys listening to music, watching movies, and running in his leisure time.
+Ehis is a Software Engineering student at Semicolon Africa.
+
+He is passionate about problem-solving.
+
+He enjoys listening to music, watching movies,running and shopping online.
```

Comment:
@ehizman, I mean you write your bio this way:

Ehis is a Software Engineering student at Semicolon Africa. He is passionate about problem-solving. He enjoys listening to music, watching movies,running and shopping online.